src/transport/raw/MessageHeader.cpp

```diff
@@ -57,6 +57,8 @@
  *
  **********************************************/
 
+#undef DEBUG
```

Comment:
Why is this here?  This should not be here.

Reply:
addressed in latest